common/data_refinery_common/models/models.py

```diff
@@ -366,6 +364,29 @@ def save(self, *args, **kwargs):
         self.last_modified = current_time
         return super(ComputationalResultAnnotation, self).save(*args, **kwargs)
 
+
+class Pipeline(models.Model):
+    "Pipeline that is associated with a series of ComputationalResult records."""
+
+    name = models.CharField(max_length=255)
+    steps = ArrayField(models.IntegerField(), default=[])
```

Comment:
Ideally we would want to have something like this instead:
```
steps = ArrayField(
        models.ForeignKey(ComputationalResult, on_delete=models.CASCADE),
        default=[]
) 
```
But it is not allowed in Django, see:
https://docs.djangoproject.com/en/2.0/ref/contrib/postgres/fields/#django.contrib.postgres.fields.ArrayField.base_field

So I am using the regular integer to refer to the primary key of related ComputationalResult record.